src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -855,24 +872,26 @@ uint32_t PolicyHandler::ChooseRandomAppForPolicyUpdate(
 
 void PolicyHandler::OnDeviceSwitching(const std::string& device_id_from,
                                       const std::string& device_id_to) {
-  LOG4CXX_AUTO_TRACE(logger_);
   POLICY_LIB_CHECK_VOID();
-  policy_manager_->OnDeviceSwitching(device_id_from, device_id_to);
+  CallPolicyManagerFunction(
+      &PolicyManager::OnDeviceSwitching, device_id_from, device_id_to);
 }
 
 void PolicyHandler::OnGetStatusUpdate(const uint32_t correlation_id) {
-  LOG4CXX_AUTO_TRACE(logger_);
   POLICY_LIB_CHECK_VOID();
+  std::string policy_table_status =
```

Comment:
@SNiukalov Const?

Reply:
@AByzhynar 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/2614/commits/98dc17dda0378940c016a6b9a16f1e3992ee8e65